build/csslint/xfail.list

```diff
@@ -87,10 +88,13 @@ shared/style/confirm.css 7 0
 shared/style/contacts/contacts_buttons.css 0 7
 shared/style/date_selector.css 1 0
 shared/style/dialer/dialer.css 0 1
+shared/style/edit_mode.css 0 2
 shared/style/edit_mode.css 1 0
```

Comment:
It was already there in the document, I didn't add it, I just sort the file so it ended here.

Still, as you said the latter rule override the first, so it isn't needed anymore